Mage.Sets/src/mage/cards/t/Triskelion.java

```diff
@@ -30,10 +30,16 @@ public Triskelion(UUID ownerId, CardSetInfo setInfo) {
         this.toughness = new MageInt(1);
 
         // Triskelion enters the battlefield with three +1/+1 counters on it.
-        this.addAbility(new EntersBattlefieldAbility(new AddCountersSourceEffect(CounterType.P1P1.createInstance(3)), "with three +1/+1 counters on it"));
-
-        // Remove a +1/+1 counter from Triskelion: Triskelion deals 1 damage to any target.
-        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new DamageTargetEffect(1), new RemoveCountersSourceCost(CounterType.P1P1.createInstance()));
+        this.addAbility(new EntersBattlefieldAbility(
+                new AddCountersSourceEffect(CounterType.P1P1.createInstance(3)),
+                "with three +1/+1 counters on it"
+        ));
+
+        // Remove a +1/+1 counter from Triskelion: It deals 1 damage to any target.
+        Ability ability = new SimpleActivatedAbility(
+                new DamageTargetEffect(1).setText("it deals 1 damage to any target"),
```

Comment:
Can use auto-generation here. See additional params of the effect. `setText` method should be used as little as possible (someday there are will be verify tests to collect stats of `setText` usages).

![shot_230902_052549](https://github.com/magefree/mage/assets/8344157/c0bcd576-f14f-492e-be65-361f6b3c377a)

Reply:
Oh missed that.
How about having a `setReplaceThisInRule(String replacement)` for stuff like that? That way we set {this} in the rule, and if set at the end of getRules we replace any {this} by the replacement text? On a per-effect basis, no need to have all effect have that method.
I feel like we don't really want rule logic in the effect constructor (they do not modify anything), and could use it there for the "sacrifice it" from the Phantasmal creatures.

Reply:
Ok I've pushed my proposal on that for the `SacrificeSourceEffect` text generation.
@xenohedron mentioned on Discord they have opinions on the matter but no time right now to express them, so let's wait on discussion to settle before merging this one.